Sources/Configuration/Versioning.xcconfig

```diff
@@ -1,2 +1,2 @@
 CURRENT_PROJECT_VERSION = 1
-MARKETING_VERSION = 8.49.1
+MARKETING_VERSION = 8.49.2
```

Comment:
`h`: Something is off here. Here we have `8.49.2` and sometimes we have `8.49.3`, but we already released 8.50.0. All versions should already be `8.50.0`.

Reply:
I haven't merged from `main` since 8.50.0 was released, and I did a find/replace on some of these I think, so I'll get the versions straightened out here and for the other ones you mentioned.